src/helpers/references.ts

```diff
@@ -78,3 +87,31 @@ export function splitReference(ref: string): { sheetName?: string; xc: string }
 export function getFullReference(sheetName: string | undefined, xc: string): string {
   return sheetName !== undefined ? `${getCanonicalSheetName(sheetName)}!${xc}` : xc;
 }
+
+/** Convert an RC notation to an A1 notation */
+export function toA1(reference: string, currentPosition?: CellPosition): string {
+  const { xc, sheetName } = splitReference(reference);
+  return `${sheetName ? sheetName + "!" : ""}${xc
```

Comment:
`sheetName` might require quotes if there's a space (or some special character) in the name.
Check `getFullReference`